pkg/users/local.js

```diff
@@ -47,6 +47,24 @@ const sortGroups = groups => {
     });
 };
 
+export function get_next_available_id(already_used, is_groups, change) {
+    //  Handle also the case where logindef == null, i.e. the file does not exist.
+    //  While that's unusual, "empty /etc" is a goal, and it shouldn't crash the page.
+    cockpit.file("/etc/login.defs").read()
+            .then(logindef => {
+                const ids = already_used.filter(a => a.name !== 'nobody').map(a => is_groups ? a.gid : a.uid);
```

Comment:
Similar issue as ; why is `nobody` special here? This should already be covered by the [UG]ID_MAX checks; (also, for groups it would be wrong anyway, it's `nogroup`). I.e. it shoudl be fine to just drop that filter()